src/core/test/fuzzing/src/test/scala/Fuzzing.scala

```diff
@@ -123,40 +175,48 @@ trait SerializationFuzzing[S <: PipelineStage with MLWritable] extends FuzzingMe
       val loadedStage = reader.load(path)
       (stage, loadedStage) match {
         case (e1: Estimator[_], e2: Estimator[_]) =>
-          assert(compareDFs(e1.fit(fitDF).transform(transDF), e2.fit(fitDF).transform(transDF)))
+          assert(e1.fit(fitDF).transform(transDF) === e2.fit(fitDF).transform(transDF))
         case (t1: Transformer, t2: Transformer) =>
-          assert(compareDFs(t1.transform(transDF), t2.transform(transDF)))
+          assert(t1.transform(transDF) === t2.transform(transDF))
         case _ => throw new IllegalArgumentException(s"$stage and $loadedStage do not have proper types")
       }
       ()
     } finally {
-      FileUtils.forceDelete(new File(path))
+      try{
+        FileUtils.forceDelete(new File(path))
+      }catch{
+        case _: FileNotFoundException =>
```

Comment:
was this meant to be ignored?  Maybe we should at least print something in this case to warn the user that the delete failed?

Reply:
This effectively tries to delete a file if it exists and if not it doesent worry about it. making it into a try catch allows the previous error message to be surfaced